eclipse-theia/theia-blueprint
# Theia 1.20.0, yarn upgrade

#### What it does

Update Blueprint's underlying framework, Theia, to v1.20.0 and run `yarn upgrade` to update various dependencies to the latest possible, as per their requested version range. 



__Vulnerabilities__ (`yarn audit`):

Before this PR:
```
48 vulnerabilities found - Packages audited: 1846
Severity: 14 Low | 31 Moderate | 3 High
```

After this PR (yarn upgrade):
```
28 vulnerabilities found - Packages audited: 1709
Severity: 14 Low | 14 Moderate
```

#### How to test

Build and run `Blueprint`, both in-place and in packaged form. Make sure everything still seems to work.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
